config.toml

```diff
@@ -153,11 +153,22 @@ contentDir = "content/ko"
 time_format_blog = "2006.01.02"
 language_alternatives = ["en"]
 
+[languages.ja]
+title = "Kubernetes"
+description = "Production-Grade Container Orchestration"
+languageName = "日本語 Japanese"
+weight = 4
```

Comment:
Please make sure these weights and file contents are in sync with the current version of `config.toml` in `master`. If accepted right now, this change would create a conflict with other languages.

Reply:
- Duplicate weights seem to be `Norsk` which is set to a disable language.
  Can I change this Norsk to be the last weight?
- Is it better to change the way to merge `master` into `dev-1.13-ja.1` and edit `config.toml` ?

Reply:
> Can I change this Norsk to be the last weight?

Yes. 👍 

> Is it better to change the way to merge master into dev-1.13-ja.1 and edit config.toml ?

That sounds correct. The goal is for `dev-1.13-ja.1#config.toml` to base on the most current version of `master#config.toml`.

Reply:
I fixed it.